src/esp/assets/PTexMeshData.cpp

```diff
@@ -616,29 +618,41 @@ void PTexMeshData::uploadBuffersToGPU(bool forceReload) {
                         Magnum::GL::MeshIndexType::UnsignedInt);
   }
 
+  // load atlas data and upload them to GPU
+  LOG(INFO) << "loading atlas textures: ";
   for (size_t iMesh = 0; iMesh < renderingBuffers_.size(); ++iMesh) {
-    const std::string rgbFile =
-        atlasFolder_ + "/" + std::to_string(iMesh) + "-color-ptex.rgb";
-    if (!io::exists(rgbFile)) {
-      ASSERT(false, "Can't find " + rgbFile);
-    }
-    LOG(INFO) << "\rLoading atlas " << iMesh + 1 << "/"
-              << renderingBuffers_.size() << "... ";
-    LOG(INFO).flush();
+    const std::string hdrFile = Cr::Utility::Directory::join(
+        atlasFolder_, std::to_string(iMesh) + "-color-ptex.hdr");
+
+    CORRADE_ASSERT(io::exists(hdrFile), "Error : Cannot find the .hdr file ", );
```

Comment:
@mosra :
For the 2nd parameter, I cannot do something like, "Error : Cannot find the .hdr file " << hdrFile as the compiler returns large number of errors.

Reply:
Does it complain also if you `#include <Corrade/Utility/DebugStl.h>`?

Reply:
I did include a header file, but maybe not this one. Let me try. Thanks.